content/documentation/distributed-tracing/index.adoc

```diff
@@ -20,51 +20,40 @@ toc::[]
 
 Kiali shows tracing of your applications. We currently support Jaeger version 1.9 and newer (link:https://www.jaegertracing.io/docs/1.12/frontend-ui/#embedded-mode[Read docs, window="_blank"]).
 
-icon:bullhorn[size=2x]{nbsp} To embed Jaeger, you must have the same protocol in Kiali and Tracing service.
+icon:bullhorn[size=2x]{nbsp} To have the tracing integration, you must set the configuration in external services / tracing.
```

Comment:
This line is a little bit vague to me. Should we add a link to the affected lines on the kiali CR? Or should we have a solid section in this page saying how to enable tracing features? (e.g. 1. edit the kiali cr, 2. look for the internal jaeger url, 3.). Just thinking out loud. We might reduce the number of questions in the FAQ page.

Reply:
I agree with @xeviknal we could be more accurate. Tracing integration is supposed to work out of the box in our nominal case so perhaps the user won't have to do anything.

Also, I would remove this text from the "icon:bullhorn" section (just plain paragraph).
What about something along these lines:

```
In most cases, the traces integration in Kiali should be enabled by default without any specific configuration. Kiali will look for the `istio-tracing` service in namespace `istio-system`. However if your setup differs from this default, you would have to provide a specific configuration.

Note also that configuring an external URL for Jaeger will also enable more links from Kiali to Jaeger UI.
```

Then, follow with the configuration details